DataStructures.Tests/Heap/MinMaxHeapTests.cs

```diff
@@ -9,13 +9,20 @@ namespace DataStructures.Tests.Heap;
 [TestFixture]
 public static class MinMaxHeapTests
 {
-    private static readonly object[] CollectionsSource =
+    private static readonly object[] CollectionsSourceInt =
```

Comment:
What is the reason to split it into multiple sources? It used to work with a single source

Reply:
Because there was an error with the generic type but I went back to see what it was and it worked so I kept it the way it was